### DISCUSSION TITLE
Unable to fetch result from OAuth login

### DISCUSSION THREAD

**Discussion by Medfac9** [Web3Auth/Web3Auth#446] [Help]
Hello everyone! i am trying to login with Web3Auth and JWT and am getting this error. Can anybody help me?
Thank you very much

![image](https://user-images.githubusercontent.com/20267230/185872991-fa28cc42-238d-4b6f-aefb-19f8b3dbd93b.png)

**Comment by YZhenY** [comment]
Can you provide the console log errors?

**Reply by Medfac9** [reply]
Yes, of course

![image](https://user-images.githubusercontent.com/20267230/186082977-ee80c48e-c490-4f76-9c22-ce4c0be321e7.png)

**Comment by chaitanyapotti** [comment]
Looks like your third party cookies are disabled. Please enable and try again

**Reply by Medfac9** [reply]
Thank you for your reply, but third party cookies are enabled

**Comment by shahbaz17** [comment]
An issue was created related to this and I have answered https://github.com/Web3Auth/Web3Auth/issues/457#issuecomment-1226746516

Let us know if this helps.

**Reply by shahbaz17** [reply]
@Medfac9

**Comment by shahbaz17** [comment]
Duplicate https://github.com/Web3Auth/Web3Auth/discussions/515
